### PR TITLE
Added req: [analog_2] for ports which requires it

### PR DESCRIPTION
**Pull request by Fraxinus88**
Added req: [analog_2] for some of my ports which requires it

### CODE REVIEW

**Comment by JeodC** [comment]
Do these definitely need *two* sticks, or is *one* permissible?

**Comment by Fraxinus88** [comment]
Definitely need twin sticks for these 4 games :)

**Comment by Slayer366** [comment]
I agree with adding this, but kloptops would be more suited to approving this particular addition to the port.json structure than I.

File: ports/forgodden/port.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "rtr": true,
     "exp": false,
     "runtime": null,
-    "reqs": [],
+    "reqs": [analog_2],
```
Comment (kloptops) [inline ports/forgodden/port.json:25]:
```suggestion
    "reqs": ["analog_2"],
```

File: ports/nulldrifter/port.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "rtr": false,
     "exp": false,
     "runtime": null,
-    "reqs": [],
+    "reqs": [analog_2],
```
Comment (kloptops) [inline ports/nulldrifter/port.json:23]:
```suggestion
    "reqs": ["analog_2"],
```

File: ports/superraftboatclassic/port.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "rtr": false,
     "exp": false,
     "runtime": null,
-    "reqs": [],
+    "reqs": [analog_2],
```
Comment (kloptops) [inline ports/superraftboatclassic/port.json:23]:
```suggestion
    "reqs": ["analog_2"],
```

**Review by kloptops — CHANGES_REQUESTED** [review]
Needs fixes. 😬

**Comment by Fraxinus88** [comment]
Okies, committed the changes :)

**Review by kloptops — APPROVED** [review]

**Pull request merged**